<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPartDefault</class>
 <widget class="QDialog" name="AddPartDefault">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar padrões</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/images/images/Objectif-photo-Logo-1.png</normalon>
   </iconset>
  </property>
  <widget class="QDialogButtonBox" name="bot_ok">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>500</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bot_add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>505</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bot_clear">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>505</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
  <widget class="QLabel" name="PartDef_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PartDef_8">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>350</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PartDef_9">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>350</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PartDef_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PartDef_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PartDef_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PartDef_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>6</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>PEÇAS PADRÕES SALVAS</string>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="PartDef_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PartDef_1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>6</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>PEÇAS PADRÕES CAPTURADAS</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="NameDef_1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>45</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameDef_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>45</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameDef_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>45</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameDef_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>185</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameDef_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>185</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameDef_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>185</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameDef_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>325</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameDef_8">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>325</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameDef_9">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>325</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="PartCap_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PartCap_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PartCap_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumDef_6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>185</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumDef_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>185</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumDef_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>185</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="PartCap_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PartCap_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PartCap_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumDef_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>45</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumDef_7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>325</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="PartCap_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PartCap_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumDef_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>45</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumDef_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="PartCap_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumDef_9">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>325</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumDef_8">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>325</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>bot_ok</zorder>
  <zorder>bot_add</zorder>
  <zorder>bot_clear</zorder>
  <zorder>PartDef_5</zorder>
  <zorder>PartDef_8</zorder>
  <zorder>PartDef_9</zorder>
  <zorder>PartDef_6</zorder>
  <zorder>PartDef_7</zorder>
  <zorder>PartDef_2</zorder>
  <zorder>PartDef_3</zorder>
  <zorder>PartDef_4</zorder>
  <zorder>PartDef_1</zorder>
  <zorder>NameDef_1</zorder>
  <zorder>NameDef_2</zorder>
  <zorder>NameDef_3</zorder>
  <zorder>NameDef_4</zorder>
  <zorder>NameDef_5</zorder>
  <zorder>NameDef_6</zorder>
  <zorder>NameDef_7</zorder>
  <zorder>NameDef_8</zorder>
  <zorder>NameDef_9</zorder>
  <zorder>PartCap_2</zorder>
  <zorder>PartCap_1</zorder>
  <zorder>PartCap_3</zorder>
  <zorder>NumDef_6</zorder>
  <zorder>NumDef_4</zorder>
  <zorder>NumDef_5</zorder>
  <zorder>PartCap_4</zorder>
  <zorder>PartCap_6</zorder>
  <zorder>PartCap_5</zorder>
  <zorder>NumDef_3</zorder>
  <zorder>NumDef_7</zorder>
  <zorder>PartCap_7</zorder>
  <zorder>PartCap_8</zorder>
  <zorder>NumDef_2</zorder>
  <zorder>NumDef_1</zorder>
  <zorder>PartCap_9</zorder>
  <zorder>NumDef_9</zorder>
  <zorder>NumDef_8</zorder>
 </widget>
 <tabstops>
  <tabstop>bot_add</tabstop>
  <tabstop>bot_clear</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bot_ok</sender>
   <signal>accepted()</signal>
   <receiver>AddPartDefault</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bot_ok</sender>
   <signal>rejected()</signal>
   <receiver>AddPartDefault</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
